src/tdastro/astro_utils/passbands.py

```diff
@@ -0,0 +1,249 @@
+import logging
+import os
+import socket
+import urllib.request
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+
+import numpy as np
+import scipy.integrate
+
+from tdastro.sources.physical_model import PhysicalModel
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+
+
+class Passbands:
+    """Class to handle the transmission tables and normalization for different passbands.
+
+    Attributes
+    ----------
+    bands : list
+        A list of strings representing the band names.
+    data_path : str
+        Path to the directory where the bandpass data is stored.
+    transmission_tables : dict
+        A dictionary with band names as keys and transmission tables as values. Transmission tables
+        are 2D arrays with wavelengths (Angstrom) in the first column and transmission strengths in
+        the second.
+    normalized_system_response_tables : dict
+        A dictionary with band names as keys and normalized system response tables as values. These
+        tables are 2D arrays with wavelengths (Angstrom) in the first column and normalized system
+        response values in the second.
+    """
+
+    def __init__(self, bands=None):
+        if bands is None:
+            self.bands = ["u", "g", "r", "i", "z", "y"]
```

Comment:
I'm not sure we need default values assuming LSST. Maybe it's better to just let users to define what bands they need, or at least print out a message to tell them you are now using these filters if you don't define them yourself.

Reply:
Default removed!